ENIGMAsystem/SHELL/Platforms/General/PFthreads.h

```diff
@@ -18,16 +18,22 @@
 #ifndef ENIGMA_PLATFORM_THREADS_H
 #define ENIGMA_PLATFORM_THREADS_H
 
-#if defined(_WIN32) || defined(__WIN32__) || defined(_WIN64) || defined(__WIN64__)
-#include <windows.h>
+#include "Universal_System/var4.h"
+
+#ifdef ENIGMA_PTFRM_SDL
```

Comment:
I don't care much for this; why not parameterize `template<typename pf_thread_t> struct ethread`? There may be a good reason, such as "various default functions require knowledge of this type"... but I get the feeling those are few and easy to work around.